src/pages/index.jsx

```diff
@@ -1,95 +1,138 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
 import Card from '@mui/material/Card'
-import CardActions from '@mui/material/CardActions'
 import CardContent from '@mui/material/CardContent'
 import CardMedia from '@mui/material/CardMedia'
+import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography'
 import Head from 'next/head'
 import Image from 'next/image'
 
-import orcas from '../../public/images/orca-breach.jpg'
+import arrow from '../../public/images/Arrow.png'
+import orcas from '../../public/images/homepage.png'
+import vector from '../../public/images/vector.png'
 import homeStyles from '../styles/Home.module.css'
-import ActionButton from './../components/ActionButton.jsx'
+
 
 export const index = () => {
   return (
-    <div className={homeStyles.home}>
+    <>
       <Head>
-        <title>Orcasound liste live for orcas</title>
+        <title>Orcasound | listen live for orcas</title>
```

Comment:
For consistency, would recommend a separate PR for typo-fixes.

Reply:
@evanjscallan good call on keeping PRs narrowly scoped, I generally agree that PRs should do one thing/feature. In this case though, IMO as a reviewer I actually wouldn't mind typo fixes in here. It's kinda arguably in scope—since the whole page is being updated to match the designs. It's really about the reason for the rule in the first place, which is to make PRs readable for easy reviewing.

Reply:
That's just my personal preference/opinion though, and of course a separate PR rarely hurts (although if it's a single line change it's almost more work to review/merge a separate PR—even though it's _technically_ the right thing to do… rules are meant to be broken after all)